hack/rpm-deps.sh

```diff
@@ -93,6 +93,12 @@ libvirtdevel_extra="
   lz4-libs
 "
 
+sandbox_base="
+  gcc
+  python36
+  glibc-static
```

Comment:
Nit: other package lists are sorted alphabetically, and so should this one.

Reply:
Done. The problem is that "someone" started introducing alphabetical orders here. :dagger: 

Done. :)

Reply:
If that's something that's considered undesirable in KubeVirt, I'll try to avoid it in the future... It's very much a habit though, so I can't guarantee it won't happen again :see_no_evil: